### PR TITLE
Add Modal component

### PR DESCRIPTION
**Pull request by SleepDevil**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/xiejay97/react-devui/blob/main/CONTRIBUTING.md#commit-message-guidelines)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: packages/ui/src/styles/components/_modal.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@include b(modal) {
+  @include e(body) {
```
Comment (kagol) [inline packages/ui/src/styles/components/_modal.scss:2]: b/e这两个函数是什么意思呢？

Comment (xiejay97) [reply packages/ui/src/styles/components/_modal.scss:2]: BEM规范

Comment (SleepDevil) [reply packages/ui/src/styles/components/_modal.scss:2]: 就是这个pr我看还没合并，是有什么地方需要补充嘛？

Comment (xiejay97) [reply packages/ui/src/styles/components/_modal.scss:2]: 之前不是和你微信说了吗，你确定这是已完成的代码吗 🤔

**Pull request closed without merging**